pele_platform/Adaptive/simulation.py

```diff
@@ -22,21 +22,18 @@
 import pele_platform.Adaptive.box as bx
 import pele_platform.Adaptive.solvent as sv
 import pele_platform.Adaptive.pca as pca
-# import pele_platform.RNA.prep as pr
 
 
 def run_adaptive(args: pv.EnviroBuilder) -> pv.EnviroBuilder:
     """
     Main function to prepare and launch simulation
 
-    1) Crate working folders and inputs
+    1) Create EnviroBuilder - variables, folders, logger...
     2) Prepare ligand and receptor
     3) Launch simulation
     4) Analyse simulation
     """
 
-    # Build Folders and Logging and env variable that will contain
```

Comment:
Why are you removing the few comments that our code has? 😢 Are they misleading?

Reply:
Point 1) above covers the same thing! :)